code/modules/hydroponics/hydroponics.dm

```diff
@@ -27,7 +27,7 @@
 	var/unwrenchable = 1
 	var/recent_bee_visit = FALSE //Have we been visited by a bee recently, so bees dont overpollinate one plant
 	var/using_irrigation = FALSE //If the tray is connected to other trays via irrigation hoses
-	var/self_sufficiency_req = 20 //Required total dose to make a self-sufficient hydro tray. 1:1 with earthsblood.
+	var/self_sufficiency_req = 20 //Required total dose to make a self-sufficient hydro tray. 1:1 with adminoriazine.

```

Comment:
You do know that adminordrazine already has an effect when used on plant trays, right?

Also, it's spelled "adminordrazine", not "adminoriazine".

Reply:
Yes, but its effect doesnt really add any self-sufficiency to the plants. Its more for when the admins want to make a meme plant tray.